aioftp/server.py

```diff
@@ -972,7 +973,7 @@ def __init__(self,
         connection = Connection(
             client_host=host,
             client_port=port,
-            server_host=self.server_host,
+            server_host=server_host or self.server_host,
```

Comment:
I see. That is because you start server as `python -m aioftp`, default value for `server_host` is `None` in `server.py`. But in `__main__.py` it is `0.0.0.0`. I think this can be changed to `None` in `__main__.py` and `server_host=self.server_host or writer.transport.get_extra_info("sockname")[0]` logic in `server.py`.